test/e2e/operator_test.go

```diff
@@ -380,7 +380,7 @@ func TestUniqueDomainRejection(t *testing.T) {
 //
 // TODO: should this be a test of source IP preservation in the conformance suite?
 func TestProxyProtocolOnAWS(t *testing.T) {
-	if infraConfig.Status.Platform != configv1.AWSPlatformType {
+	if infraConfig.Status.PlatformStatus.Type != configv1.AWSPlatformType {
```

Comment:
The tests currently pass but `PlatformStatus` can be `nil`.

Reply:
I added a nil check.